pkg/process/cache.go

```diff
@@ -40,9 +41,15 @@ var colorStr = map[int]string{
 	deleted:       "deleted",
 }
 
-// garbage collection run interval
 const (
+	// garbage collection run interval
 	intervalGC = time.Second * 30
+	// garbage collection stale entry check interval.
+	// We will check for stale entries once per interval.
+	staleIntervalGC = time.Hour * 1
+	// garbage collection stale entry max time.
+	// If a stale entry is older than the max time, then it will be removed.
+	staleIntervalMaxTime = time.Minute * 10
```

Comment:
> > I agree. If we are doing this, it would also be nice to have a switch to configure the parameters that right now are hardcoded.
> 
> And also make the existing intervalGC value configurable? It's been hard-coded for ever.

I think we meant those ones. Or at least the first one `staleIntervalGC`